packages/zowe-explorer/src/trees/dataset/DatasetTree.ts

```diff
@@ -233,12 +233,6 @@ export class DatasetTree extends ZoweTreeProvider<IZoweDatasetTreeNode> implemen
             if (!node) continue;
             const srcDsn = SharedUtils.getNodeProperty(node, "label");
 
-            // 1. Same-object check
-            if (node.getProfile && target.getProfile && (await SharedUtils.isSamePhysicalDataset(node.getProfile(), target.getProfile(), srcDsn))) {
-                Gui.errorMessage(vscode.l10n.t(SharedUtils.ERROR_SAME_OBJECT_DROP));
-                return;
-            }
```

Comment:
Agree with @awharn that we want to keep the prompt associated with this check, and add a button to allow users to bypass/override the same-DASD warning.